File: ArrayAndList/min_max/min_max.py
arr=[12,3,45,65,1]
max=arr[0]
min=arr[0]
for i in range(len(arr)):
    if arr[i]>max:
        max=arr[i]
    elif arr[i]<min:
        min=arr[i]
print("max:",max)
print("min:",min)

File: ArrayAndList/min_max/min_max.js
let arr=[12,32,4,5,3,111];
let max=arr[0];
let min=arr[0];
for(let i=0;i<arr.length;i++){
    if(arr[i]>max) max=arr[i];
    else if(arr[i]<min) min=arr[i];
}
console.log("max:",max);
console.log("min;",min);